sciencebeam_utils/tools/split_csv_dataset.py

```diff
@@ -63,6 +69,55 @@ def split_rows(rows, percentages, fill=False):
     ]
 
 
+def _to_hashable(value):
+    return str(value)
```

Comment:
Do you need this function?

Reply:
I am adding `rows` to a set. A `row` will in reality be an array (but in tests may be a string or a number). I only use the set to check whether they exist, i.e. I don't need a reverse. Here calling `str` which will format an array as something like `["value1", "value2"]` seems to do. But I could call join, calculate a hash or do anything similar. The faster the better. But it's not used that often and doesn't need to be that optimised if it's fast enough.

Where I use the function I don't care about the details, which is why I have a separate function.

Reply:
Okay, I've switched to using `tuple` and changed the tests to always pass in the rows with the correct type. I've also added type hints (comments for Python 2).